spec/lib/phobos_spec.rb

```diff
@@ -28,4 +28,18 @@
       expect(Phobos.create_exponential_backoff).to be_a(ExponentialBackoff)
     end
   end
+
+  describe '.logger' do
+    context 'with "config.logger.file" defined' do
```

Comment:
Ok, I agree. I can add a new test that checks if we create the file and maybe the subdirectories, ex: `/var/logs/my-consumer/phobos.yml`